paddle/fluid/inference/tests/api/int8_mkldnn_quantization.md

```diff
@@ -59,12 +70,22 @@ Notes:
 cd /PATH/TO/PADDLE/build
 python ../paddle/fluid/inference/tests/api/full_ILSVRC2012_val_preprocess.py
 ```
-The converted data binary file is saved by default in ~/.cache/paddle/dataset/int8/download/int8_full_val.bin
+Then the ILSVRC2012 Validation dataset will be preprocessed and saved by default in ~/.cache/paddle/dataset/int8/download/int8_full_val.bin.
    * ##### ResNet50 Full dataset benchmark
 ```bash
 ./paddle/fluid/inference/tests/api/test_analyzer_int8_resnet50 --infer_model=third_party/inference_demo/int8v2/resnet50/model --infer_data=/path/to/converted/int8_full_val.bin --batch_size=1 --paddle_num_threads=1
 ```
-   * ##### Mobilenet-v1 Full dataset benchmark
+   * ##### Other 6 models Full dataset benchmark
 ```bash
-./paddle/fluid/inference/tests/api/test_analyzer_int8_mobilenet --infer_model=third_party/inference_demo/int8v2/mobilenet/model --infer_data=/path/to/converted/int8_full_val.bin --batch_size=1 --paddle_num_threads=1
+./paddle/fluid/inference/tests/api/test_analyzer_int8_resnet50 --infer_model= --infer_data=/path/to/converted/int8_full_val.bin --batch_size=1 --paddle_num_threads=1
 ```
+Set argument `--infer_model` with the following values in command line for other 6 models' benchmark:
```

Comment:
`for other 6 models' benchmark:` -> `to benchmark 6 other models:`

Reply:
Done